### PR TITLE
Updated README.md with addition of Flutter

### PR DESCRIPTION
**Pull request by abhisen-exe**
thanks for reading my pull request!

### CODE REVIEW

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]
Add `Flutter resources in Cross platform section
To create a Cross-platform section into the index in alphabetical order use :point_down: 
```
- [Cross Platform Dev](#crossplatformdev)
  * [Flutter](#flutter)
```

Then create a section of Cross-platform for resources too :point_down: 
```
## Cross Platform Dev

### Flutter
```

Please make sure to add the section in alphabetical order

**Comment by abhisen-exe** [comment]
The requested changes are done!

File: README.md [hunk 1]
```diff
@@ -207,7 +207,15 @@ Add resources in other languages too :point_down:
 - :bulb: [FAUN Publication](https://faun.dev/c/)
 
 ---
+
+
```
Comment (Ashish-khanagwal) [inline README.md:211]: remove this extra empty line `211`

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]
under line `70` (`[Core-CS](#core-cs)`) add the following code :point_down: 
```
- [Cross Platform Dev](#crossplatformdev)
  * [Flutter](#flutter)
```

**Pull request closed without merging**